examples/celeba_glow.py

```diff
@@ -0,0 +1,612 @@
+#! -*- coding: utf-8 -*-
+# Keras implement of Glow
+# https://blog.openai.com/glow/
+
+from keras.layers import *
+from keras.models import Model
+from keras.datasets import cifar10
+from keras import backend as K
+from keras.callbacks import Callback
+from keras.optimizers import Adam
+import imageio
+import numpy as np
+from scipy import misc
+import glob
+import os
+
+if not os.path.exists('samples'):
+    os.mkdir('samples')
+
+imgs = glob.glob('img_align_celeba/*.jpg')
+
+height, width = misc.imread(imgs[0]).shape[:2]
+center_height = int((height - width) / 2)
+
+img_size = 64  # for a fast try, please use img_size=32
+depth = 10  # orginal paper use depth=32
+level = 3  # orginal paper use level for 256*256 CelebA HQ
+
+
+def imread(f):
```

Comment:
Since this function does more than reading an image.

`keras.preprocessing.image.load_img` already does something similar.

I suggest instead introducing a `normalize_img` function that does `img / 256 - 0.5`, and chaining it with `load_img`

Reply:
I am not very familiar with keras's utils part. because it does not appear in https://keras.io 
I think they are some simple operations, so I do that by myself.